source/cloud/vault/connecting.md

```diff
@@ -1,4 +1,5 @@
 ```eval_rst
+.. title: UKFast Documentation | eCloud Vault | Connecting to eCloud Vault
```

Comment:
its the meta title that goes in  <title>xxx</title> looking to change that seems linked to the title set by # as above example at the moment..